config/aggregator/vector.yaml

```diff
@@ -0,0 +1,52 @@
+---
+data_dir: /vector-data-dir
+api:
+  enabled: true
+  address: "[::]:8686"
+  playground: false
+sources:
+  datadog_agent:
+    address: "[::]:8282"
+    type: datadog_agent
+  fluent:
+    address: "[::]:24224"
+    type: fluent
+  internal_metrics:
+    type: internal_metrics
+  logstash:
+    address: "[::]:5044"
+    type: logstash
+  splunk_hec:
+    address: "[::]:8080"
+    type: splunk_hec
+  statsd:
+    address: "[::]:8125"
+    mode: tcp
+    type: statsd
+  syslog:
+    address: "[::]:9000"
+    mode: tcp
+    type: syslog
+  vector:
+    address: "[::]:6000"
+    type: vector
+    version: "2"
+sinks:
```

Comment:
@zsherman @jszwedko what do you think about defaulting with a `pipelines` transform with a couple of commented out pipelines to serve as examples? Without defined pipelines, data would just pass through, so the configuration would still work. The rationale is that we nudge more users towards `pipelines` transform for simplified processing.

Reply:
@spencergilbert spoke to @zsherman about this. Let's add a `pipelines` transform that all sources fan into. It can be empty for now. I'll think of default pipelines that we can add or comment.

Reply:
👍 I'll figure out pipeline transforms work 😆

Reply:
@binarylogic @zsherman, talking with @jszwedko - since `pipelines` haven't been released, is it appropriate to have as a part of our default config?

Reply:
Yeah, agreed, I think we should leave out pipelines until they are released and documented.

Reply:
Removed the defaults from this PR based on discussion with @jszwedko